averages.js

```diff
@@ -0,0 +1,66 @@
+const pry = require('pryjs')
+
+class Operations {
+    constructor(collection) {
+        this.collection = collection
+    }
+
+    get total() {
+        return this.collection.reduce(function(sum, value) {
+            return sum + value;
+        }, 0)
+    }
+
+    get mean() {
+        return this.total / this.collection.length
+    }
+
+    get median() {
+
+        let sorted = this.collection.sort(function(a, b) { a - b })
```

Comment:
Remember that `Array.prototype.sort` is mutative, so you'll often see a pattern like this: `arr.slice(0).sort(...)`